src/Objects/Player.gd

```diff
@@ -56,3 +57,10 @@ func _physics_process(delta: float) -> void:
 		is_jumping = true
 
 	velocity = move_and_slide(velocity, Vector2.UP, true)
+
+
+func bounce():
+	if Input.is_action_pressed("ui_up"):
+		velocity.y = -JUMP_POWER
```

Comment:
This line will be replaced with a better line that calculates how high it should be, but that doesn't need to be part of your PR